MAP/src/App.js

```diff
@@ -234,11 +240,45 @@ const App = (props) => {
        setEnvironment(envt);
       }
   }
+  const updateMapCord = (value) =>{
+    const cord = value.split("/");
+    if(cord.length === 3){
+      setMapCord((prevCord) => ({
+        ...prevCord,
+        lng: cord[0],
+        lat: cord[1],
+        zoom: cord[2]
+      }));
+    }
+  }
+  const opeOverlay = async (value) => {
+      const countryIso  = await fetchCountryISO();
+      if(countryIso.length){
+        const selectedCountry =await _.find(countryIso, {"Iso": value});
+        if(selectedCountry){
+          setDialog(prevState => ({
+            ...prevState,
+            opened: true,
+            template: '',
+            title: '',
+            iso2: value,
+            country: selectedCountry.name,
+          }
+        ));
+        }
+      }
+  }
     
     useEffect(() =>{
       const {search=""} = props.location;
       const params = new URLSearchParams(search);
       for (const [key, value] of params) {
+        if(key === "map"){
+          updateMapCord(value);
+        }
+        else if(key==="PLD"){
```

Comment:
these can't be used together :thinking: maybe they might be someday....(i don't know)